pkg/controller/node/nodecontroller.go

```diff
@@ -544,6 +548,63 @@ func (nc *NodeController) monitorNodeStatus() error {
 	return nil
 }
 
+// monitorNodeTaints for all nodes compares taints with pods tolerations
+// and sends pods for eviction
+func (nc *NodeController) monitorNodeTaints() error {
+	nodes, err := nc.kubeClient.Core().Nodes().List(api.ListOptions{})
```

Comment:
Because this is a net new function, it feels like this should be informer driven, not polling list driven.